be/src/olap/cumulative_compaction_time_series_policy.cpp

```diff
@@ -63,6 +63,12 @@ uint32_t TimeSeriesCumulativeCompactionPolicy::calc_cumulative_compaction_score(
         return 0;
     }
 
+    auto consecutive_empty_rowsets = tablet->pick_first_consecutive_empty_rowsets(
+            tablet->tablet_meta()->time_series_compaction_empty_rowsets_threshold());
```

Comment:
how about no threshold in meta

Reply:
When constructing TabletMeta, the threshold is assigned a default value of 5. If the threshold is not present in the meta, it implies that there was an issue during the construction of the meta.